src/apyanki/note.py

```diff
@@ -608,6 +718,8 @@ def markdown_file_to_notes(filename: str) -> list[NoteData]:
                 fields=x["fields"],
                 markdown=x["markdown"],
                 deck=x["deck"],
+                nid=x.get("nid"),
+                cid=x.get("cid"),
```

Comment:
Since we already defined the defaults, we can use `x["nid"]` and `x["cid"]` here. I think the consistency with the above is nice.